astroplan/plots/sky.py

```diff
@@ -0,0 +1,204 @@
+
+    style_kwargs : dict or Empty, optional.
+        A dictionary of keywords passed into `matplotlib.pyplot.scatter`
+        to set plotting styles.
+
+    north_to_east_ccw: bool, optional.
+        True by default, meaning that azimuth is shown increasing
+        counter-clockwise (CCW), or with North at top, East at left, etc.
+        To show azimuth increasing clockwise (CW), set to False.
+
+    grid: bool, optional.
+        True by default, meaning that grid is drawn.
+
+    az_label_offset: `~astropy.units.degree`, optional.
+        DANGER: It is not recommended that you change the default behavior,
+        as to do so makes it seem as if N/E/S/W are being decoupled from the
+        definition of azimuth (North from az = 0 deg., East from az = 90 deg.,
+        etc.).
+        An offset for azimuth labels from the North label.  A positive
+        offset will increase in the same direction as azimuth
+        (see `north_to_east_ccw` option).
+
+    Returns
+    -------
+    An `Axes` object (ax) with a map of the sky.
+
+    Notes
+    -----
+    Coordinate defaults:
+
+        Altazimuth (local horizon) coordinate system.  North is always at top
+        of plot, South is always at the bottom, E/W can be right or left
+        depending on the `north_to_east_cw` option.
+
+        Altitude: 90 degrees (zenith) is at plot origin (center) and 0 degrees
+        (horizon) is at plot edge.  This cannot be changed by user.
+
+        Azimuth: 0 degrees is at North (top of plot), 90 degrees at East, etc.
+        DANGER: Azimuth labels can be changed by user via the `az_label_offset`
+        option, but it is not recommended, as to do so makes it seem as if
+        N/E/S/W are being decoupled from the definition of azimuth
+        (North from az = 0 deg., East from az = 90 deg., etc.).
+
+
+    TODO:
+        1) Add time/date and target name labels?
+    """
+
+    # Set up axes & plot styles if needed.
+    if ax is None:
+        # axisbg option sets background color within plot bounds (circle).
+        # May need to set a color outside of plot bounds.
+        ax = plt.gca(projection='polar')
+        #ax = plt.gca(polar=True)
+    if style_kwargs is None:
+        style_kwargs = {}
+    style_kwargs = dict(style_kwargs)
+    style_kwargs.setdefault('marker', 'o')
+
+    # Populate time window if needed.
+    time = Time(time)
+    if time.isscalar:
+        time = time + np.linspace(-12, 12, 25)*u.hour
+
+    # Grab altitude and azimuth from Astroplan objects.
+    # Note that values must be made dimensionless before plotting.
+    # Modifying altitude is easier than inverting r-axis.
+    altitude = (91 * u.deg - observer.altaz(time, target).alt) *(1/u.deg)
+    # Azimuth MUST be given to plot() in radians.
+    azimuth = observer.altaz(time, target).az * (1/u.deg) * (np.pi/180.0)
+
+    # Some checks & info for labels.
+    if not hasattr(target, 'name'):
+        target_name = '?'
```

Comment:
I put a `?` here because of [the warning](https://github.com/astroplanners/astroplan/pull/30/files#diff-01da251e6d4193d7d339fe7f85713f14R127) that is spit out when a user tries to plot positions that are below the horizon.